service-api/src/main/java/greencity/constant/ErrorMessage.java

```diff
@@ -55,5 +55,5 @@ public class ErrorMessage {
         "You do not have permission to deactivate this user";
     public static final String BASE64_DECODE_MESSAGE = "Can't decode from base64 format";
     public static final String BRUTEFORCE_PROTECTION_MESSAGE =
-        "User account is blocked due to too many failed login attempts";
+        "User account is blocked due to too many failed login attempts. Try again in an hour";
```

Comment:
I see that the default value is 1 hour
`bruteForceSettings.blockTimeInHours=${BLOCK_TIME_IN_HOURS:1}`
Do we use other values?
if yes we should build a message based on them or check when the blocking time ends

Reply:
I see that logic is based on user IP, not name
Should we modify a message to represent current logic?